ecosystem/html_templates/webpage.html.jinja

```diff
@@ -73,28 +76,99 @@
 
     <div class="content cds--grid layout-lead-space-fixed__container">
       <h2 id="providers" class="section">
```

Comment:
Sounds good to stick with this URL. That makes it easier since we already have some active links to this.

Reply:
Great! Thank you Eric!